official/vision/beta/projects/yolo/modeling/backbones/darknet.py

```diff
@@ -88,41 +96,40 @@ def build_block_specs(config):
   return specs
 
 
-class LayerFactory(object):
-  """Class for quick look up of default layers.
-
-  Used by darknet to connect, introduce or exit a level. Used in place of an if
-  condition or switch to make adding new layers easier and to reduce redundant
-  code.
+class LayerFactory:
```

Comment:
Forgive my pedantry, but I don't think "Factory" is the right name for the pattern used in this class. Could be confusing to future readers. Perhaps a "LayerBuilder"? (https://en.wikipedia.org/wiki/Builder_pattern)

Reply:
Renamed to "LayerBuilder"